schemes/rrtmgp/rrtmgp_inputs.F90

```diff
@@ -254,33 +245,50 @@ subroutine rrtmgp_inputs_run(dosw, dolw, snow_associated, graupel_associated, &
      ! to be consistent with t_sfc.
      emis_sfc(:,:) = 1._kind_phys
 
+     !-------------------------------------------------------------------------
+     ! RRTMGP enforces  P > 1 Pa for validity.
+     ! In radiation.F90 we count layers based on P_ref > 10 Pa to safely account
+     ! for possible situations in MPAS (z-based vert. coordinate) in which
+     ! full 3D pressure could be significanlty below min(P_ref).
+     ! (we really shoud have a variable called 'p_top_for_rrtmgp')
+     !
+     ! If 
+     ! 1) entire vertical domain has P_ref> 10Pa (e.g. CAM7 LT) then
+     !    nlay = pverp
+     !    ktoprad = 2
+     !    ktopcam = 1
+     ! 2) min(P_ref) < 10Pa (e.g. CAM7 MT) then
+     !    nlay < pverp
+     !    ktoprad = 1
+     !    ktopcam = pver - nlay + 1
+     !
      ! Level ordering is the same for both CAM and RRTMGP (top to bottom)
+     ! Note in Case 2, tops of {t,pint,pmid}_rad start at a 'valid' level,
+     ! i.e. pref > p_top_for_rrtmgp, given by ktopcam ...
```

Comment:
I don't think (?) the `given by ktopcam` is needed here anymore, as `p_top_for_rrtmgp` is now an input argument.